IOSAccessAssessment/IOSAccessAssessmentApp.swift

```diff
@@ -9,9 +9,21 @@ import SwiftUI
 
 @main
 struct IOSAccessAssessmentApp: App {
+    private let keychainService = KeychainService()
+    @State private var isAuthenticated: Bool
+    
+    init() {
+        let isTokenValid = keychainService.isTokenValid()
+        _isAuthenticated = State(initialValue: isTokenValid)
```

Comment:
So mostly, I do not see a problem with this kind of initialization for this context, despite its misgivings given in:
https://stackoverflow.com/questions/56691630/swiftui-state-var-initialization-issue

More specifically: https://stackoverflow.com/a/71247040

I wonder if there would come a situation where this initialization could cause problems. Most likely it shouldn't. 
I'll just keep it in mind.

Reply:
Just as a note, we don't have a problem with this way to setting the State because the struct IOSAccessAssessment does not have any dependencies.

Reply:
But this article does mention that this is bad practice, since we are accommodating what may lead to potentially inconsistent behavior:
https://www.swiftcraft.io/articles/how-to-initialize-state-inside-the-views-init-